docs/essentials/introduction.md

```diff
@@ -110,4 +111,4 @@ For example, if you wanted to disable the [backgrounds addon](./backgrounds.md),
 
 💡 You can use the following keys for each individual addon: `actions`, `backgrounds`, `controls`, `docs`, `viewport`, `toolbars`, `measure`, `outline`.
```

Comment:
@winkerVSbecks if you wouldn't mind updating this sentence and adding this addon as well I'd appreciate it so that we can mention to our new/existing users that they can disable it by providing the necessary key!

Reply:
Marking this as solved as the sentence was updated per feedback.